tasks/libs/dynamic_test/indexers/e2e.py

```
@@ -137,52 +89,37 @@ def _extract_relative_path(self, file_path: str) -> str:
         # Build path starting from after github.com/DataDog/datadog-agent, i.e., join from index 3
         return "/".join(segments[3:])
 
-    def _parse_coverage_line(self, line: str) -> tuple[str, bool]:
+    def parse_coverage_line(self, line: str) -> tuple[str, str, bool]:
```

This part his highly coupled with the coverage format. Should we raise an Exception somewhere if we detect a change in such format (even if it should not change often)